tutor/src/helpers/images-complete.js

```diff
@@ -0,0 +1,22 @@
+export default function imagesComplete(body) {
+
+  return new Promise((resolve) => {
+    const images = body.querySelectorAll('img');
+    let complete = 0;
+    const markComplete = () => {
+      complete += 1;
+      if (complete === images.length) {
+        return resolve(images);
+      }
+    };
+    Array.from(images).forEach((img) => {
+      if (img.naturalWidth) {
+        markComplete();
```

Comment:
Aha! Nice.

Reply:
Yes, I didn't really know about that either until I googled it a bit - is a nice way to check if an image is loaded